### DISCUSSION TITLE
Hottop and Phidget

### DISCUSSION THREAD

**Discussion by chammoud** [artisan-roaster-scope/artisan#1075] [Q&A]
I'm new to roasting but it is becoming my relaxation time to experiment with roasting profiles over the weekend.  I'm been using Hottop roaster with Artisan.  I'd like to integrate phidget thermocouple module to my roast.  has anyone integrated Hottop and phidget data in the same plotting routine of Artisan.  Is it doable?  if so how?
Thank you a bunch.

Happy roasting!!

Chafic

**Comment by MAKOMO** [comment]
Just add an extra device and select the type of Phidget you connect.

**Reply by chammoud** [reply]
Out of curiosity, is there any setup direction to have Artisan negotiate Hottop roaster and the phidget thermocouple module in the same plotting routine during the roast process?

**Reply by poundy** [reply]
Chafic, I assume English is not your native language. I suspect you're asking how to get this extra thermocouple to be displayed on the graph, when roasting??

I have no experience with a hottop to know what sensors it has. As Marko says, you simply need to enable an extra device, that is your Phidget connected temp probe, via the Config -> Device -> Extra Devices and/or Config -> Device -> Phidgets dialogs.

**Reply by chammoud** [reply]
Thank you for the info.  We'll give it a try.
